homeassistant/components/uptimerobot/binary_sensor.py

```diff
@@ -27,6 +29,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     up_robot = UptimeRobot()
     api_key = config.get(CONF_API_KEY)
     monitors = up_robot.getMonitors(api_key)
+    id_prefix = config.get(CONF_PREFIX)
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler